src/Storefront/Resources/app/storefront/src/plugin/google-analytics/events/view-item.event.js

```diff
@@ -28,20 +42,4 @@ export default class ViewItemEvent extends AnalyticsEvent
             }],
         });
     }
-
-    /**
-     * @param { NodeList } inputs
-     * @return ?string
-     */
-    findProductId(inputs) {
```

Comment:
Please also mention in the changelog that this was removed. 

@PaddyS It this supposed to be extendable by plugins?

Reply:
@tobiasberge I added it to the changelog. I also wrote the suggestion what to do, if that is needed.

I think this is a complicated, potentially error prone way of determining the `productId`.

Reply:
Thank you, changelog looks good to me. 👍